projects/flow.hoomd.lj/src/init.py

```diff
@@ -3,60 +3,38 @@
 
 Iterates over all defined state points and initializes
 the associated job workspace directories."""
-import logging
-import argparse
-from hashlib import sha1
-
 import signac
 import numpy as np
 
 
-def main(args, random_seed):
+def main():
     project = signac.init_project('Ideal-Gas-Example-Project')
-    for replication_index in range(args.num_replicas):
-        for p in np.linspace(0.5, 5.0, 10):
-            statepoint = dict(
-                    # system size
-                    N=512,
-
-                    # Lennard-Jones potential parameters
-                    sigma=1.0,
-                    epsilon=1.0,
-                    r_cut=2.5,
-
-                    # random seed
-                    seed=random_seed*(replication_index + 1),
-
-                    # thermal energy
-                    kT=1.0,
-                    # presure
-                    p=p,
-                    # thermostat coupling constant
-                    tau=1.0,
-                    # barostat coupling constant
-                    tauP=1.0)
-            project.open_job(statepoint).init()
+    for p in np.linspace(0.5, 5.0, 10):
```

Comment:
Maybe add a code comment here, something like "sweep over 10 different pressures from 0.5 to 5.0 at 0.5 pressure increments " not sure on the exact wording but people might not know exactly what np.linspace does